File: 0001 Code Chef/0004 Ides of march/implementation/0009 dart/ides_of_march.dart
import 'dart:io';


void main(){
  String? stringInput = stdin.readLineSync();
  int? day = int.tryParse(stringInput!);
  if(day != null){
    if(day == 15){
      print("YES");
    }else{
      print("NO");
    }
  }
}

File: 0001 Code Chef/0004 Ides of march/implementation/0010 nodejs/ides_of_march.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});

rl.question('',(input)=>{
    day = parseInt(input);
    if(day == 15){
        console.log("YES");
    }else{
        console.log("NO");
    }
    rl.close()
});